src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -215,6 +188,18 @@ void PacketDataReporter::OnCommissionableNodeIPAddress(const chip::Inet::IPAddre
     mCommissionableNodeData.ipAddress[mCommissionableNodeData.numIPs++] = addr;
 }
 
+bool HasQNamePart(SerializedQNameIterator qname, QNamePart part)
```

Comment:
Please place helper functions in anonymous namespace (so that they are static to the compilation unit and compiler can optimize if possible)

Reply:
I forgot this entire class is in anon namespace (was used to placing anon on top outside class methods). Ignore my comment here.